typespec-tests/src/main/java/com/azure/resourcemanager/models/resources/fluent/models/TopLevelTrackedResourceInner.java

```diff
@@ -85,7 +85,6 @@ public SystemData systemData() {
      * 
      * @return the type value.
      */
-    @Override
     public String type() {
```

Comment:
Is this change expected? They are getters of the shadowed read-only properties from parent class.

Reply:
Oh, I saw it's added back in later commits!